keras/src/backend/jax/distribution_lib.py

```diff
@@ -118,72 +118,64 @@ def distribute_data_input(inputs, layout):
             `jax.sharding.Sharding` instance.
 
     Returns:
-        Distributed inputs thats been properly put to local devices.
+        Distributed inputs that's been properly put to local devices.
```

Comment:
A global batch distributed according to `layout`.

or something like that?

Reply:
Done!